platform-emf/src/main/java/com/softicar/platform/emf/management/importing/engine/EmfImportEngine.java

```diff
@@ -0,0 +1,143 @@
+package com.softicar.platform.emf.management.importing.engine;
+
+import com.softicar.platform.common.core.exceptions.SofticarUserException;
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.db.runtime.field.IDbField;
+import com.softicar.platform.db.sql.field.ISqlField;
+import com.softicar.platform.emf.EmfI18n;
+import com.softicar.platform.emf.attribute.field.transaction.EmfTransactionAttribute;
+import com.softicar.platform.emf.deactivation.IEmfTableRowDeactivationStrategy;
+import com.softicar.platform.emf.table.IEmfTable;
+import com.softicar.platform.emf.table.row.IEmfTableRow;
+import com.softicar.platform.emf.token.parser.EmfTokenMatrixParser;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class EmfImportEngine<R extends IEmfTableRow<R, P>, P, S> {
+
+	private final IEmfTable<R, P, S> table;
+	private final List<List<String>> textualRows;
+	private final List<R> parsedRows;
+	private final Set<ISqlField<?, ?>> ignoredFields;
+	private Optional<S> scope;
+
+	public EmfImportEngine(IEmfTable<R, P, S> table) {
+
+		this.table = table;
+		this.textualRows = new ArrayList<>();
+		this.parsedRows = new ArrayList<>();
+		this.ignoredFields = new HashSet<>();
+		this.scope = Optional.empty();
+
+		ignoreActiveField();
```

Comment:
I wonder if, instead of using this `ignoredFields` approach, there should be `EmfTokenMatrixParser` options to ignore certain field types (active, generated, transaction, scope).
Maybe based upon generic `IDbField` predicates.
This seems kind of misplaced here... what do you think?

Reply:
I think this should be something that the user can override, at least the active field.